src/api/APIFactory.js

```diff
@@ -592,6 +603,20 @@ class APIFactory {
         this.boxEditAPI = new BoxEditAPI();
         return this.boxEditAPI;
     }
+
+    /**
+     * API for Annotations
+     *
+     * @return {AnnotationsAPI} AnnotationsAPI instance
+     */
+    getAnnotationsAPI(shouldDestroy: boolean): AnnotationsAPI {
```

Comment:
Is it a good idea to add a default value?

Reply:
Yeah, good idea.